### PR TITLE
Polls view changes

### PR DESCRIPTION
**Pull request by DariaSova**

### CODE REVIEW

File: app/views/layouts/_flash.html.haml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- flash.each do |name, msg|
+  - if name.eql?("notice")
+    - name = "success"
+  = content_tag :div, msg, class: "alert-" + name
```
Comment (Sinetheta) [inline app/views/layouts/_flash.html.haml]: Doesn't this class need to be `alert alert-whatever`? http://getbootstrap.com/components/#alerts

Comment (DariaSova) [inline app/views/layouts/_flash.html.haml]: yep, good call! that makes the alert box wider

File: app/helpers/devise_helper.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module DeviseHelper
+  def devise_error_messages!
+    return '' if resource.errors.empty?
+
+    messages = resource.errors.full_messages.map { |msg| content_tag(:li, msg) }.join
+    html = <<-HTML
```
Comment (Sinetheta) [inline app/helpers/devise_helper.rb]: Since there's not much HTML here, how about a regular old content_tag?

**Comment by Sinetheta** [comment]
:ship:

**Pull request merged**